packages/app-store/alby/components/AlbyPaymentComponent.tsx

```diff
@@ -137,22 +134,17 @@ function PaymentChecker(props: PaymentCheckerProps) {
         if (props.booking.status === "ACCEPTED") {
           return;
         }
-        const bookingsResult = await utils.viewer.bookings.get.fetch({
-          filters: {
-            status: "upcoming",
-            eventTypeIds: [props.eventType.id],
-          },
+        const bookingsResult = await utils.viewer.bookings.find.fetch({
```

Comment:
Is it possible to update this to a subscription so the polling can be removed?

Reply:
We don't have something for subscription, we can use trpc but still revalidates and it will be the same as the polling we are doing right now, probably we can increase the base interval and stop it after 60 seconds?

Reply:
I don't think it would be good to stop polling so quickly. A user might have trouble paying with one wallet and try another for example. But I guess we could slowly ramp up the delay after multiple polls though?